store/postgres/migrations/2021-05-21-165219_add_created_at_columns_for_deployment_schemas/up.sql

```diff
@@ -0,0 +1,5 @@
+alter table public.deployment_schemas
+    add column created_at timestamptz not null default now();
+
+alter table public.unused_deployments
+    add column deployment_created_at timestamptz not null;
```

Comment:
@lutter I'm not sure how should we handle default values in this case.

For migrations applied on a running instance, this migration will probably fail due to missing values.
Should we allow this column to contain null values?

Reply:
Yes, that was my intent - use a value that is clearly made up in case we ever need to tell real values from fake ones.